pytorch_lightning/trainer/trainer.py

```diff
@@ -243,8 +244,15 @@ def __init__(
             num_sanity_val_steps: Sanity check runs n validation batches before starting the training routine.
                 Set it to `-1` to run all batches in all validation dataloaders. Default: 2
 
+            reload_dataloaders_every_n_epochs: Set to a non negative integer to reload dataloaders every n epochs.
+                Default: 0
```

Comment:
Default should be 1

Reply:
Gotcha, I'll update it. Previously `reload_dataloaders_every_epoch` was set to `False` by default so I kept it at 0

Reply:
Ok, the default should be 0 and please add 0 means never reload.